modules/ROOT/pages/introduction-monitoring-metrics.adoc

```diff
@@ -77,3 +106,49 @@ However, the MicroProfile Metrics feature doesn't provide metrics about Open Lib
 If you enable both the MicroProfile Metrics and Performance Monitoring features, then the MXBeans for monitoring and the `/metrics` REST endpoint are activated.
 In this case, metrics for Open Liberty server components are exposed through both interfaces.
 The MicroProfile Metrics feature version 2.3 and later automatically enables the Performance Monitoring feature.
+
+==  Integration with Micrometer
+
+The Open Liberty feature for MicroProfile Metrics 5.0, {mp-5-0}, provides integration with the Micrometer metrics technology to leverage its capability to transfer metric data to multiple third-party monitoring systems. There are 18 different third-party metric monitoring systems that Micrometer can be connected to. See a full list at the https://micrometer.io/docs[Micrometer metrics documentation]. This integration is achieved by providing to the Open Liberty runtime the Micrometer core library, Micrometer registry implementations and its dependencies. This configuration strategy does not provide access to the Micrometer metrics API in the application.
+
+By default, the {mp-5-0} feature can only provide support for the Prometheus monitoring tool through the `/metrics` endpoint. To customize the monitoring systems that the {mp-5-0} can transfer metric data to, provide the Micrometer core libraries, Micrometer registry implementations and all necessary dependencies by defining a https://openliberty.io/docs/latest/reference/config/library.html[Shared Library] in the `server.xml`. The shared library specified is then referenced by the `libraryRef` attribute of the {mp-5-0} configuration element. It is important to note that there will be no `/metrics` endpoint unless the Micrometer Prometheus registry library is provided as part of the https://openliberty.io/docs/latest/reference/config/library.html[Shared Library].
+
+The user-provided Micrometer registry implementations need to be enabled by setting a corresponding `mp.metrics.<registry>.enabled` property to `true` before metric data can be sent to the associated  metric monitoring tool. For example, if the Elastic Micrometer registry library and its dependencies are provided, the `mp.metrics.elastic.enabled` property must be set to `true` to render it active. This is true for all Micrometer registry implementations except for the Prometheus registry implementation where the default behavior is to have it enabled if available. The `<registry>` value is an identifying value designated for each individual Micrometer registry. Review the https://micrometer.io/docs[Micrometer metrics documentation] to identify the respective values for each Micrometer registry.
```

Comment:
Hm, I guess we're not allowed to link to the docs of another project? Otherwise people might not know how to get to the right `<registry>` value to use. I guess the alternative is to make a table?